docs/api/1.x/buckets.rst

```diff
@@ -25,7 +25,7 @@ Creating a bucket
 
     .. sourcecode:: bash
 
-        $ echo '{"data": {"id": "blog"}}' | http POST http://localhost:8888/v1/buckets --auth="bob:" --verbose
+        $ echo '{"data": {"id": "blog"}}' | http POST http://localhost:8888/v1/buckets --auth="token:bob" --verbose
```

Comment:
That's to help readability of the tutorial, would you suggest that:
 - we use some random token?
 - we replace ``bob`` with ``bob-token``?